### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by amarjandu**
found missing commas, and double quote.

### CODE REVIEW

File: daemons/dss-notify-v2/Readme.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Bundle creation notifications have the format
 
 ```
 {
-  'dss_api': {dss_api_url}
-  'bundle_url': {dss_api}"/v1/bundles/{uuid}?version={version}
+  'dss_api': {dss_api_url},
```
Comment (DailyDreaming) [inline daemons/dss-notify-v2/Readme.md]:
```
'dss_api': "{dss_api_url}",
'bundle_url': "{dss_api_url}/v1/bundles/{uuid}?version={version}",
```

Comment (amarjandu) [reply daemons/dss-notify-v2/Readme.md]: from what i see the other parts of this readme, strings are double quote wrapped. variables are {} wrapped.

Comment (DailyDreaming) [reply daemons/dss-notify-v2/Readme.md]: The two main points:

`dss_api_url` and `dss_api` are the same thing, so should have the same name.

`{dss_api}"/v1/bundles/{uuid}?version={version}"` doesn't make any sense.

Comment (amarjandu) [reply daemons/dss-notify-v2/Readme.md]: updated.

Comment (amarjandu) [reply daemons/dss-notify-v2/Readme.md]: updated.

**Pull request merged**